index.html

```diff
@@ -0,0 +1,162 @@
+          <a href="#">
+            <span>itunes</span>
+          </a>
+        </li>
+        <li class="support">
+          <a href="#">
+            <span>support</span>
+          </a>
+        </li>
+      </ul>
+      <div class="search">
+        <form action="#" method="post">
+          <div class="input">
+            <input type="text" name="search">
+          </div>
+        </form>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section class="hero">
+      <a class="full" href="#">What will your verse be?</a>
+      <header class="header">
+				<h1 class="title">
+					<a href="#">
+						<img class="large" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_large.png" width="980" height="134" alt="What will your verse be?">
+						<img class="medium" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_medium.png" width="980" height="112" alt="What will your verse be?">
+						<img class="small" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small.png" width="980" height="104" alt="What will your verse be?">
+					</a>
+				</h1>
+				<h2 class="subheadline">
+					<a href="#">
+						<img class="large" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_large.png" width="980" height="176" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+						<img class="medium" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_medium.png" width="980" height="86" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+						<img class="small" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png" width="980" height="82" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+					</a>
+				</h2>
+				<div class="center">
+					<a href="#">
+						<img class="medium" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_27pt.png" width="980" height="64" alt="Watch the “Your Verse” film and explore the stories">
+						<img class="large" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_46pt.png" width="980" height="112" alt="Watch the “Your Verse” film and explore the stories">
+					</a>
+				</div>
+			</header>
+    </section>
+  </main>
+  
+  <div class="promo">
+    <ul>
+      <li>          
+        <a class="promo1" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" width="364" height="200" alt="iPhone 5s. Forward thinking.">
```

Comment:
**width="364" height="200"**
Add the right selectors for width and height in your CSS file